tests/disaster-recovery/sc_arbiter/test_netsplit.py

```diff
@@ -0,0 +1,450 @@
+        time.sleep((duration + 5) * 60)
+        end_time = datetime.now(timezone.utc)
+        logger.info(f"Ended netsplit at {end_time}")
+
+        # wait for the logreader workload to finish
+        statuses = ["Completed"]
+        if zones in ("bc", "ab-bc"):
+            statuses.append("Error")
+
+        wait_for_pods_to_be_in_statuses(
+            expected_statuses=statuses,
+            pod_names=[pod.name for pod in logreader_pods],
+            timeout=900,
+            namespace=constants.STRETCH_CLUSTER_NAMESPACE,
+        )
+        logger.info("Logreader job pods have reached 'Completed' state!")
+
+        # check if all the read operations are successful during the failure window, check for every minute
+        if check_for_read_pause(logreader_pods, start_time, end_time):
+            logger.info(f"Read operations are paused during {zones} netsplit window")
+        else:
+            logger.info("All the read operations are successful!!")
+
+        # check if all the write operations are successful during the failure window, check for every minute
+        for i in range(len(logwriter_pods)):
+            try:
+                if check_for_write_pause(
+                    logwriter_pods[i], log_file_map.keys(), start_time, end_time
+                ):
+                    logger.info(
+                        f"Write operations paused during {zones} netsplit window"
+                    )
+                else:
+                    logger.info("All the write operations are successful!!")
+                    break
+            except CommandFailed as e:
+                if (
+                    "Permission Denied" in e.args[0]
+                    or "unable to upgrade connection" in e.args[0]
+                ) and zones in ["bc", "ab-bc"]:
+                    continue
+                else:
+                    assert (
+                        False
+                    ), f"{logwriter_pods[i].name} pod failed to exec command with the following eror: {e.args[0]}"
+
+        # reboot the nodes where the pods are not running
+        pods_not_running = [
+            pod
+            for pod in logwriter_pods
+            if OCP(
+                kind="Pod", namespace=constants.STRETCH_CLUSTER_NAMESPACE
+            ).get_resource_status(pod.name)
+            != constants.STATUS_RUNNING
+        ]
+        assert (
+            zones in ("bc", "ab-bc") or len(pods_not_running) == 0
+        ), f"Unexpectedly these pods {pods_not_running} are not running after the {zones} failure"
+
+        for pod in pods_not_running:
+            node_obj = get_pod_node(pod)
+            nodes.stop_nodes(nodes=[node_obj], wait=False)
+            wait_for_nodes_status(
+                node_names=[node_obj.name], status=constants.NODE_NOT_READY
+            )
+            nodes.start_nodes(nodes=[node_obj])
+            wait_for_nodes_status(node_names=[node_obj.name])
+
+        wait_for_pods_to_be_in_statuses(
```

Comment:
you have used this func in many places with a change in pod name. You can actually create a separate function for it 
def wait_for_pods_completion(pods, timeout=TEST_TIMEOUT):